app_flutter/lib/commit_box.dart

```diff
@@ -33,15 +33,32 @@ class _CommitBoxState extends State<CommitBox> {
 
   @override
   Widget build(BuildContext context) {
+    int authorHash = widget.commit.author.hashCode;
     return SizedBox(
       width: StatusGrid.cellSize,
       height: StatusGrid.cellSize,
       child: GestureDetector(
         onTap: _handleTap,
+        // TODO(chillers): Show a Network Image. https://github.com/flutter/flutter/issues/45955
+        // CanvasKit currently cannot render a NetworkImage because of CORS issues.
+        // As a work around, we just show the first letter of the contributor's username.
```

Comment:
Unrelated to code: could you do a PSA in the infra discord channel to explain this change when deployed?

Reply:
SGTM